eg/Utils.py

```diff
@@ -394,23 +394,23 @@ def IsVista():
     """
     warnings.warn(
         "eg.Utils.IsVista() is deprecated. "
-        "Use eg.WindowsVersion.IsVista() instead",
+        "Use eg.WindowsVersion >= 'Vista' instead",
         DeprecationWarning,
         stacklevel=2
     )
-    return eg.WindowsVersion.IsVista()
+    return eg.WindowsVersion >= 'Vista'
 
 def IsXP():
     """
     Determine if we're running XP or higher.
     """
     warnings.warn(
         "eg.Utils.IsXP() is deprecated. "
-        "Use eg.WindowsVersion.IsXP() instead",
+        "Use eg.WindowsVersion >= 'XP' instead",
```

Comment:
well the original eg.Utils.IsVista() and eg.Utils.IsXP() would return true if the os was >= vista or xp. hence the reason why i only did the >= so this way it will mimic the exact operation of the original functions. where as the new version only returns  true if it is that exact os.

if you look at the original code
    
    def IsVista():
        """
        Determine if we're running Vista or higher.
        """
        return (sys.getwindowsversion()[0] >= 6)

    def IsXP():
        """
        Determine if we're running XP or higher.
        """
        return (sys.getwindowsversion()[0:2] >= (5, 1))

even the doc string says and higher,


I'll do whatever it is you want me to do on this. but i just wanted to have to depreciation warning show exactly how to use the new to replace the old. no confusion that way